sale_rental/data/rental_data.xml

```diff
@@ -12,6 +12,7 @@
     <field name="sequence">100</field>
     <field name="warehouse_selectable" eval="True" />
     <field name="product_selectable" eval="False" />
+    <field name="company_id" ref="base.main_company" />
```

Comment:
@BT-rmartin Re-reading this, I'm not sure this is the good solution.

In fact, rental routes should have been managed through warehouse configurations.